frontend/src/old-pages/Clusters/Clusters.tsx

```diff
@@ -27,44 +25,53 @@ import {
   Table,
   TextFilter
 } from "@awsui/components-react";
-
 import { useCollection } from '@awsui/collection-hooks';
 
-// Components
 import EmptyState from '../../components/EmptyState';
 import Status from "../../components/Status";
 import Actions from './Actions';
 import Details from "./Details";
 import { wizardShow } from '../Configure/Configure';
 
-function updateClusterList() {
+
+export interface Cluster {
+  cloudformationStackArn: string,
+  cloudformationStackStatus: string,
+  clusterName: string,
+  clusterStatus: string,
+  region: string,
+  version: string
+}
+
+async function updateClusterList() {
   const selectedClusterName = getState(['app', 'clusters', 'selected']);
   const oldStatus = getState(['app', 'clusters', 'selectedStatus']);
 
-  ListClusters((clusterList: any) => {
-    if(selectedClusterName)
-    {
-      const selectedCluster = findFirst(clusterList, (c: any) => c.clusterName === selectedClusterName);
-      if(selectedCluster)
-      {
-        if(oldStatus !== selectedCluster.clusterStatus)
+  try {
+    const clusterList = await ListClusters();
+    if(selectedClusterName) {
+      const selectedCluster = findFirst(clusterList, (c: Cluster) => c.clusterName === selectedClusterName);
+      if(selectedCluster) {
+        if(oldStatus !== selectedCluster.clusterStatus) {
           setState(['app', 'clusters', 'selectedStatus'], selectedCluster.clusterStatus);
-
-        if((oldStatus === 'CREATE_IN_PROGRESS' && selectedCluster.clusterStatus === 'CREATE_COMPLETE') ||
-          (oldStatus === 'UPDATE_IN_PROGRESS' && selectedCluster.clusterStatus === 'UPDATE_COMPLETE'))
-          // @ts-expect-error TS(2554) FIXME: Expected 2 arguments, but got 1.
-          selectCluster(selectedClusterName);
+        }
+        if((oldStatus === 'CREATE_IN_PROGRESS' && selectedCluster.clusterStatus === 'CREATE_COMPLETE') || (oldStatus === 'UPDATE_IN_PROGRESS' && selectedCluster.clusterStatus === 'UPDATE_COMPLETE')) {
+            selectCluster(selectedClusterName, null);
+        }
       }
     }
+  } catch (error) {}
+}
 
-  })
+interface ClusterListProps {
```

Comment:
nitpick: `type` is preferred over `interface` for data structures - the result is the same, it's only clearer for the reader

Reply:
Thanks, going to change it!